kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/KafkaPrincipal.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.streamnative.pulsar.handlers.kop.security;
+
+
+import java.security.Principal;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+
+/**
+ * Store client login info.
+ */
+@ToString
+@AllArgsConstructor
+public class KafkaPrincipal implements Principal {
+
+    public static final String USER_TYPE = "User";
+
+    @Getter
+    private final String principalType;
+
+    /**
+     * Pulsar role.
+     */
+    @Getter
+    private final String name;
```

Comment:
Add @Getter to `KafkaPrincipal` instead of each field.

```java
@Getter
public class A {
    private final String field1;
    private final String field2;
    // ...
}
```

is better than

```java
public class A {
    @Getter
    private final String field1;
    @Getter
    private final String field2;
    // ...
}
```